main/src/com/google/refine/operations/recon/ReconCopyAcrossColumnsOperation.java

```diff
@@ -195,6 +195,6 @@ public boolean visit(Project project, int rowIndex, Row row) {
 
     @Override
     protected String getBriefDescription(Project project) {
-        return OperationDescription.recon_copy_across_columns_brief(_fromColumnName, StringUtils.join(_toColumnNames));
+        return OperationDescription.recon_copy_across_columns_brief(_fromColumnName, StringUtils.join(_toColumnNames, ", "));
```

Comment:
This was a missing separator I noticed while adapting the tests. Without this, the column names are joined without any separator.